requirements/requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
-ovos-plugin-manager~=0.0.19
-ovos-utils~=0.0.25
-streaming-stt-nemo==0.0.4a1
+ovos-plugin-manager>=0.0.19
```

Comment:
`>=` can cause problems if a breaking change is released (and properly versioned). For the ovos packages, I don't think this will install anything different; is there a particular minimum version needed?

Reply:
This was the version used originally.  it does run properly on the latest version `0.0.21a6`. Can I just leave off the version on this?

Reply:
You can update to `ovos-plugin-manager~=0.0.19,>=0.0.21a6`. This specs a minimum alpha version but prevents updating to a potentially breaking version in the future (like `1.0.0`)